packages/terra-hookshot/src/_HookshotUtils.js

```diff
@@ -12,6 +12,21 @@ const MIRROR_TB = {
   bottom: 'top',
 };
 
+/**
+ * This function returns a rect 2px x 2px with the origin based at (1,1), allow for evenly centered attachment points.
+ * As a result of the 2x2 size, bounds checking errors as a result of rounding are eliminated.
```

Comment:
`bounds checking errors as a result of rounding are eliminated.` I think you meant `bounds checking errors are eliminated as a result of rounding.`